src/design-tokens/modular-scale.yml

```diff
@@ -0,0 +1,23 @@
+aliases:
+  minor_second: 1.067
+  major_second: 1.125
+  minor_third: 1.2
+  major_third: 1.25
+  perfect_fourth: 1.333
+  augmented_fourth: 1.414
+  perfect_fifth: 1.5
+  minor_sixth: 1.6
+  golden_section: 1.618
+  major_sixth: 1.667
+  minor_seventh: 1.778
+  major_seventh: 1.875
+  octave: 2
+  major_tenth: 2.5
+  major_eleventh: 2.667
+  major_twelfth: 3
+  double_octave: 4
+props:
+  ratio:
+    value: "{!major_third}"
+    type: float
+    category: modular_scale
```

Comment:
I was confused by this file at first. I thought it was housing our actual modular scale values as opposed to the ratios. I wonder if renaming this to something like `modular-scale-ratios` would be clearer? Unless we plan to add more MS stuff here?

Reply:
There are infinite possible modular scale values based on the ratio, so I'm unsure what else we would need to store? 🤔

Reply:
I was just confused reading through this the first time since I misunderstood the purpose of this file. I thought it was going to house the scale values as opposed to the scale ratios. I thought tweaking the file name might make this clearer, but I don't think it's a big deal either way.

Reply:
Thanks for the clarification.

I think I'll keep it as-is to avoid having a `modular-scale-ratios.ratio` variable, which feels redundant to me.